src/SFA.DAS.EmployerAccounts/Queries/GetTransferConnectionInvitationAuthorization/GetTransferConnectionInvitationAuthorizationQueryHandler.cs

```diff
@@ -43,7 +43,8 @@ public async Task<GetTransferConnectionInvitationAuthorizationResponse> Handle(G
             return new GetTransferConnectionInvitationAuthorizationResponse
             {
                 AuthorizationResult = authorizationResult,
-                IsValidSender = isValidSender
+                IsValidSender = isValidSender,
+                TransferAllowancePercentage = _configuration.TransferAllowancePercentage
```

Comment:
is there a test to update to ensure this value is populated based on config

Reply:
Created two tests, one based around this handler. One based in transferallowance